app.js

```diff
@@ -1,5 +1,7 @@
 'use strict';
 
+require('newrelic');
```

Comment:
I notice this isn't mentioned in the Heroku documentation. I don't doubt that it's the right thing to do, but just to be sure I understand would you be able to say more?